packages/components/src/components/as-histogram-widget/utils/draw.service.ts

```diff
@@ -43,17 +45,21 @@ export function renderBars(
   yScale: ScaleLinear<number, number>,
   container: Container,
   barsContainer: Container,
-  BARS_SEPARATION: number,
   color: string,
   X_PADDING: number,
   Y_PADDING: number,
   disableAnimation: boolean = false) {
 
+  let BARS_SEPARATION = 1;
```

Comment:
If it's not a constant anymore, let's switch it to lowercase.
```suggestion
  let barsSeparation = 1;
```

There are more places in the file with reference to it.